WordPress/src/main/java/org/wordpress/android/ui/reader/services/discover/ReaderDiscoverLogic.kt

```diff
@@ -127,6 +130,12 @@ class ReaderDiscoverLogic constructor(private val completionListener: ServiceCom
         return cards
     }
 
+    private fun insertPostsIntoDb(posts: List<ReaderPost>) {
+        val postList = ReaderPostList()
+        postList.addAll(posts)
+        ReaderPostTable.addOrUpdatePosts(null, postList)
```

Comment:
I'm wondering if we want to insert the posts under a fake tag so we can easily delete them later. Wdyt?

Reply:
@malinajirka - why a fake tag? Couldn't they use the "Discover" readerTag ... that's what they are ? No one but this endpoint should be adding `discover` readerTag to the table anymore.

Reply:
>No one but this endpoint should be adding discover readerTag to the table anymore.

We'll need to think about this too. Discover tag was always used for posts from "discover.wordpress.com". If it's a server side tag as well we might not be able to just change it. My bet is that it's still used in Calypso's Reader.